crapssim/strategy.py

```diff
@@ -461,6 +461,26 @@ def place68_dontcome2odds(player, table, unit=5, strat_info=None):
         if not player.has_bet("LayOdds") and not dc.prepoint:
             player.bet(LayOdds(mult * unit, dc))
 
+def dontpass_2come_dontcome(player, table, unit=5, strat_info=None):
+    # start with dontpass bet
+    # once the point is established, place come bet twice
+    # finally place a dont come bet
+    # whenever there is less than 2 come bet replenish it.

```

Comment:
In looking at the strategy, I think a key feature is that you only risk 4 units per shooter.  I think this can be incorporated with a bit of extra logic.  

Additionally, it looks like they go up to 2 don't come bets.